src/client/js/pie-item.js

```diff
@@ -16,50 +16,54 @@
  *  limitations under the License
  *
  */
- (function() {
 
-  const template = document.createElement('template');
-  template.innerHTML = `
-    <pie-img width="300px" height="200px"></pie-img>
-    <div>
-      <h1 class="pie-title skeleton skeleton-on"></h1>
-      <div class="pie-description skeleton-text skeleton-on">
-        <p></p>
-        <ul>
-          <li><span class="pie-weight"></span></li>
-          <li><span class="pie-price"></span></li>
-        </ul>
-      </div>
-    </div>`;
+const template = document.createElement('template');
+template.innerHTML = `
+  <pie-img width="300px" height="200px"></pie-img>
+  <div>
+    <h1 class="pie-title skeleton skeleton-on"></h1>
+    <div class="pie-description skeleton-text skeleton-on">
+      <p></p>
+      <ul>
+        <li><span class="pie-weight"></span></li>
+        <li><span class="pie-price"></span></li>
+      </ul>
+    </div>
+  </div>`;
 
-  class PieItem extends HTMLElement {
-    constructor() {
-      super();
-    }
-    connectedCallback() {
-      let content = template.content.cloneNode(true);
-      this.appendChild(content);
-      this._connected = true;
-      if (this._pie) {
-        this._populate();
-      }
-    }
-    _populate() {
-      this.querySelector('.skeleton').classList.remove('skeleton-on');
-      this.querySelector('.skeleton-text').classList.remove('skeleton-on');
-      this.querySelector('.pie-title').innerHTML = this._pie.signature;
-      this.querySelector('.pie-weight').innerHTML = `weight: ${this._pie.weight}`;
-      this.querySelector('.pie-price').innerHTML = `price: ${this._pie.price}`;
-      this.querySelector('pie-img').setAttribute('topping', this._pie.topping);
-      this.querySelector('pie-img').setAttribute('filling', this._pie.filling);
-      this.querySelector('pie-img').setAttribute('dough', this._pie.dough);
+export default class PieItem extends HTMLElement {
+  constructor() {
+    super();
+  }
+
+  static define() {
+    window.customElements.define('pie-item', PieItem);
```

Comment:
Note: moved this here in order to avoid having the element definition happen as a side-effect upon module import. This way, it needs to be triggered explicitly by the importer.